js/source/esri-vector-tile-source.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var util = require('mapbox-gl/js/util/util');
+var Evented = require('mapbox-gl/js/util/evented');
+var TileCoord = require('mapbox-gl/js/source/tile_coord');
+var Source = require('./esri-source');
+
+module.exports = VectorTileSource;
+
+function VectorTileSource(options) {
```

Comment:
Similarly to the Pyramid, this needs to be renamed, if for no other reason to appear different than the existing VectorTileSource class in tracebacks.